vps2arch

```diff
@@ -55,6 +55,7 @@ download_and_extract_bootstrap() {
 	download iso/latest/sha1sums.txt | fgrep "$cpu_type.tar.gz" > "sha1sums.txt"
```

Comment:
I would prefer to change the fgrep pattern here instead, so we don't need to involve a separate sed call at all.